### PR TITLE
New package: stumpwm-22.11

### PR DESCRIPTION
**Pull request by funk443**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl

### CODE REVIEW

File: srcpkgs/common-lisp-clx/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'common-lisp-clx'
+pkgname=common-lisp-clx
+version=0.7.5
+revision=1
+build_style=fetch
```
Comment (Chocimier) [inline srcpkgs/common-lisp-clx/template:5]: Remove build_style variable. Then tarball is extracted by convention, so you don't need `tar -xf`, `cd`, and tar in hostmakedepends.

File: srcpkgs/common-lisp-clx/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+hostmakedepends="tar coreutils"
```
Comment (Chocimier) [inline srcpkgs/common-lisp-clx/template:6]: Coreutils are part of build environment by default, please remove.

File: srcpkgs/stumpwm/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'stumpwm'
+pkgname=stumpwm
+version=22.11
+revision=1
+build_style=gnu-configure
+hostmakedepends="sbcl base-devel"
+makedepends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
```
Comment (Chocimier) [inline srcpkgs/stumpwm/template:7]: Please add them to this PR so stumpwm builds in CI.

File: srcpkgs/stumpwm/template [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+depends="xorg common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
```
Comment (paper42) [inline srcpkgs/stumpwm/template:8]: remove xorg from dependencies, this is left to the user

File: srcpkgs/common-lisp-ppcre/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="ctoid <funk443@yandex.com>"
+license="BSD-2-Clause"
+homepage="https://edicl.github.io/cl-ppcre/"
+distfiles="https://github.com/edicl/cl-ppcre/archive/refs/tags/v${version}.tar.gz"
+checksum="89631179b71648d9e6c565a928f6896a9d5742aa2083b9c1b705fe0b45d85def"
+
+_LIBRARY_DIR=/usr/share/common-lisp/source/cl-ppcre
+
+do_install() {
+	curl -O "https://raw.githubusercontent.com/edicl/cl-ppcre/master/LICENSE"
```
Comment (Chocimier) [inline srcpkgs/common-lisp-ppcre/template:16]: Please extract license from comment, or move to distfiles with a note to use copy from tarball on nearest update.

File: srcpkgs/stumpwm/template [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'stumpwm'
+pkgname=stumpwm
+version=22.11
+revision=1
+build_style=gnu-configure
+hostmakedepends="sbcl base-devel"
+makedepends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
+depends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
```
Comment (Chocimier) [inline srcpkgs/stumpwm/template:8]: Other distros add libraries as runtime dependencies, but do anything breaks without them?

File: srcpkgs/stumpwm/template [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+short_desc="Stump Window Manager"
+maintainer="ctoid <funk443@yandex.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/stumpwm/stumpwm"
+distfiles="https://github.com/stumpwm/stumpwm/archive/refs/tags/${version}.tar.gz"
+checksum="36940f3b2c1ba54219a8d3a784103cfa83f84c652d1dbd7d3b7360c55e5ef026"
+nopie=yes
+nostrip=yes
+nocross="SBCL seems won't be able to find libraries while cross-compiling"
```
Comment (Chocimier) [inline srcpkgs/stumpwm/template:17]: More of lack of cross compilation by sbcl in general.

**Review by Chocimier — COMMENTED** [review]
Quick tests shows it works. Please fix minor issues, and I'll merge.

File: srcpkgs/stumpwm/template (hunk 6 above)
Comment (funk443) [reply srcpkgs/stumpwm/template:8]: Do you mean the libs in makedepends?

File: srcpkgs/stumpwm/template (hunk 7 above)
Comment (funk443) [reply srcpkgs/stumpwm/template:17]: What do you mean?

**Pull request closed without merging**